### PR TITLE
add events and actions for task reschedule

### PR DESCRIPTION
**Pull request by bibibox**
prepare for the volcano issue https://github.com/volcano-sh/volcano/issues/3812

### CODE REVIEW

File: pkg/apis/bus/v1alpha1/events.go [hunk 1]
```diff
@@ -30,6 +30,12 @@ const (
 	// PodEvictedEvent is triggered if Pod was deleted
 	PodEvictedEvent Event = "PodEvicted"
 
+	// PodPendingEvent is triggered if Pod was pending
+	PodPendingEvent Event = "PodPending"
+
+	// PodRunningEvent is triggered if Pod was running
+	PodRunningEvent Event = "PodRunning"
```
Comment (william-wang) [inline pkg/apis/bus/v1alpha1/events.go:37]: What's the intention for PodRunning?

Comment (bibibox) [reply pkg/apis/bus/v1alpha1/events.go:37]: When the pod corresponding to a podFailed/podEvicted event returns to the running state, its delayed action should be canceled. So we need to be able to detect the pod running event.

Comment (william-wang) [reply pkg/apis/bus/v1alpha1/events.go:37]: It makes sense:)

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**